module6/lessons/Week1/ToolsForAfterAnInterview.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: Tools For After An Interview
+---
+
+## Introduction
+
+Interviews are often a vulnerable and stressful thing. They are also a necessary hurdle on the path to getting that first developer job. It's important to gather tools for resilience to navigate this portion of the job search process.
+
+Today we are going to learn and practice two routines for after your interviews. The first is an interview recap which should feel very similar to other reflections we have done in this course, and the second is doing an emotions check-in and creating a "soft landing" for yourself.
+
+<aside class="instructor-notes" markdown="1">
+<p><strong>Instructor Note</strong><br>
+I think it would be best to do this activity in the job search stand-up groups for this week.</p>
+</aside>
+
+### What is a Soft Landing?
+
+What is a soft landing, you ask? The concept comes from [this](https://www.vickipavitt.com/blog/vulnerability-hangover) blog post titled **Nursing a Vulnerability Hangover**.
+
+Read through the blog post and then reflect on the following questions:
+
+What parts of this blog post resonate with you?
+
+How does a vulnerability hangover show up for you?
+
+What are some ways that you can nurse your vulnerability hangover so you can continue to be brave? What’s your "soft landing"?
+
+Are there ways that your group can support you in your "soft landing"? For example, maybe there is something they can remind you to do for yourself or a mantra they can remind you of.
+
+When we come back together after today's interviews, you will discuss these questions with your group and then I'll ask for volunteers to share in the full group. Then we will move into an Interview Recap and Emotional Check-In for today's interview.

```

Comment:
I think we could reframe this as 'We will have time for you to come together after each week's interview to discuss these questions'

Reply:
That way this session could be moved around :)

Reply:
Good call to make this more flexible. I'm thinking that reading the blog, individual reflection, small group sharing on these questions, and then whole group sharing is the focus of tomorrow.

And then maybe briefly going over the structure for Interview Recaps and Emotions Check-Ins. But I'm planning to guide students through the recap and check ins with a lot of structure this first time. So giving time for thinking, then going to breakout rooms, then introing next section, time for thinking, breakouts.